k8s/mainnet-singletons.yaml.j2

```diff
@@ -48,3 +48,15 @@ configmapEnvironment:
 sharedEmptyDirs:
   - name: ic-data
     mountPath: /ic-data
+
+networkPolicy:
```

Comment:
Need the net policy to talk to the shared chia nodes too

Reply:
Good callout. This should be resolved now